### ISSUE TITLE
SRIOV VirtualMachineInstance with sriov plugin interface [test_id:3957]should connect to another machine with sriov interface over IPv6

### ISSUE THREAD

**Issue by tiraboschi** [kubevirt/kubevirt#3747]
/kind bug
/triage build-officer

**What happened**:
Failed test due to timeout:
```SRIOV VirtualMachineInstance with sriov plugin interface [test_id:3957]should connect to another machine with sriov interface over IPv6```

```
/root/go/src/kubevirt.io/kubevirt/tests/vmi_multus_test.go:874
Failed to configure address fc00::2/64 for interface eth1 on VMI testvmi68rgzcj6sk8zp2x5xmnqtqgn2dd78nvpf79xc789mq9cnqpz
Unexpected error:
    <expect.TimeoutError>: 15000000000
    expect: timer expired after 15 seconds
occurred
/root/go/src/kubevirt.io/kubevirt/tests/vmi_multus_test.go:896
```

[Prow Job Link](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3684/pull-kubevirt-e2e-kind-1.17-sriov/1278748175628242945)
**Lane**:
pull-kubevirt-e2e-kind-1.17-sriov

**Comment by phoracek** [comment]
@AlonaKaplan could this be possibly caused by the expecter bug?

**Comment by AlonaKaplan** [comment]
> @AlonaKaplan could this be possibly caused by the expecter bug?

No, it's not the expecter bug.
`ip addr add fc00::2/64 dev eth1` gets an error - `Cannot find device "eth1`

**Comment by alonSadan** [comment]
hopefully #4001 will solve it.

**Comment by ormergi** [comment]
More details at #4432

**Comment by oshoval** [comment]
> More details at #4432

4432 is not related, error here is eth1 not found, on 4432 its the ping that doesnt work (when i logged in the machine i saw the device is there but ip was gone, reasons on 4432)

There is one more problem that i know of, but its not the right issue to speak about.

**Comment by ormergi** [comment]
This issue is user report for a flaky test.
This test is flaky for two reasons, one is resolved and described by https://github.com/kubevirt/kubevirt/pull/4001 
And the second is describe by https://github.com/kubevirt/kubevirt/issues/4432

**Comment by oshoval** [comment]
Fixed by #4448 by @qinqon 
/close
